src/app/application/components/route-view/RouteView.vue

```diff
@@ -21,6 +21,9 @@
             }
           )
         },
+        replace: (...args: Parameters<typeof router['push']>) => {
```

Comment:
**Suggestion**: This should be `replace`, right?

```suggestion
        replace: (...args: Parameters<typeof router['replace']>) => {
```

Reply:
Nope, we are calling `router.push` i.e. replace the current route entirely

Reply:
Ah, I got confused by the name of the method then. It should be renamed to `push` to avoid that.

Reply:
Its not push you are not pushing to a `route` you are replacing the `route` with a new one. You push to the `router`. This is the `route`.

Reply:
> Its not push you are not pushing to a route you are replacing the route with a new one. You push to the router. This is the route.

I don't think it's the same thing. The router and the route are completely different things, you push a new route to the router to make it the current route, you replace the current route with a new one to replace the current route.

Reply:
Alright. Resolving this as it’s not worth spending time on.

Reply:
Yeah my thoughts exactly.

Reply:
Is there anything else here that blocks an approval?

Reply:
There is an outstanding thread that hasn’t been addressed.